### ISSUE TITLE
[Request] Add support for aesthetic score -aided dataset into the discriminator and loss functions.

### ISSUE THREAD

**Issue by nom57** [dvschultz/stylegan2-ada-pytorch#41]
Add aesthetic-rating aided dataset support into the discriminator loss function.
Add a function that could penalize the discriminator for thinking an image from the dataset is "more real" than one which has a higher aesthetic-score rating (could be an aesthetic score system beetwen 1 to 5 or 1 to 10). We could organize a dataset based on aesthetic scores by seperating them into seperate folders (example : folders named 1,2,3,4,5,6,7,8,9,10 , for 1-10 aesthetic score rating).

This would greatly help use-cases where you have a lot of images for a dataset but only a select few are good enough in quality and creativity, but you still wanna keep the diversity of a much larger dataset.

**Comment by dvschultz** [comment]
im not really updating this repo anymore beyond managing dependencies. If anyone wants to work on this you’re welcome to and submit a PR.
